sourcegraph/sourcegraph
# Setup App: Replace custom picker API with tauri dialog package

Based on https://github.com/sourcegraph/sourcegraph/pull/51666

## Background 
As you may know, prior to this PR, we used a custom GQL handler that uses Apple script to call the native file picker/dialog UI. In this PR, we use a native picker with Tauri dialog API. 

## Test plan
- Check new local repositories setup UI page
- Check setup wizard local repositories step
- Check both UI in Mac and Windows (if you do have Windows machine)
